crypto/dilithium/pqcrystals_dilithium_ref_common/sign.c

```diff
@@ -325,6 +396,46 @@ int crypto_sign_verify(ml_dsa_params *params,
   return 0;
 }
 
+/*************************************************
+* Name:        crypto_sign_verify
+*
+* Description: FIPS 204: Algorithm 3 ML-DSA.Verify.
+*              Verifies signature.
+*
+* Arguments:   - ml_dsa_params: parameter struct
+*              - uint8_t *m: pointer to input signature
+*              - size_t siglen: length of signature
+*              - const uint8_t *m: pointer to message
+*              - size_t mlen: length of message
+*              - const uint8_t *ctx: pointer to context string
+*              - size_t ctxlen: length of context string
+*              - const uint8_t *pk: pointer to bit-packed public key
+*
+* Returns 0 if signature could be verified correctly and -1 otherwise
+**************************************************/
+int crypto_sign_verify(ml_dsa_params *params,
+                       const uint8_t *sig,
+                       size_t siglen,
+                       const uint8_t *m,
+                       size_t mlen,
+                       const uint8_t *ctx,
+                       size_t ctxlen,
+                       const uint8_t *pk)
+{
+  size_t i;
+  uint8_t pre[257];
+
+  if(ctxlen > 255)
+    return -1;
```

Comment:
Here and in other spots in this PR please update the code to always add the brackets even for single line statements. 
```suggestion
  if(ctxlen > 255) {
    return -1;
  }
```

Reply:
added in [54ca109](https://github.com/aws/aws-lc/pull/1999/commits/54ca1098fc3bc9a0629a24931e041dc72561bda6)